internal/hmyapi/types.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/harmony-one/go-sdk/pkg/address"
```

Comment:
wrong import - this will give go-sdk cyclic imports problem

Reply:
yes, corrected.